src/config/config.js

```diff
@@ -116,7 +116,7 @@ const Config = {
   textileSecret: env.REACT_APP_TEXTILE_SECRET,
   web3Polling: env.REACT_APP_WEB3_POLLING || 30 * 1000, //poll every 30 seconds by default
   realmAppID: env.REACT_APP_REALM_APP_ID || 'wallet_dev-dhiht',
-  isFVFlow: process.env.REACT_APP_BUILD_TARGET === 'FV',
+  isLoginFlow: process.env.REACT_APP_BUILD_TARGET === 'FV',
```

Comment:
no this should be called isFVFlow
@serdiukov-o-nordwhale

Reply:
yep, search - replace may be cruel :(
thanks for finding this